kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -326,158 +326,168 @@ class SchemaUtilsSuite extends AnyFunSuite {
   // computeSchemaChangesById
   ///////////////////////////////////////////////////////////////////////////
   test("Compute schema changes with added columns") {
-    val fieldMappingBefore = Map(
-      1 -> new StructField("id", IntegerType.INTEGER, true))
+    val fieldMappingBefore = newSchema((1, new StructField("id", IntegerType.INTEGER, true)))
 
-    val fieldMappingAfter = Map(
-      1 -> new StructField("id", IntegerType.INTEGER, true),
-      2 -> new StructField("data", StringType.STRING, true))
+    val fieldMappingAfter = newSchema(
+      (1, new StructField("id", IntegerType.INTEGER, true)),
+      (2, new StructField("data", StringType.STRING, true)))
 
-    val schemaChanges = schemaChangesHelper(fieldMappingBefore, fieldMappingAfter)
+    val schemaChanges = computeSchemaChangesById(fieldMappingBefore, fieldMappingAfter)
 
     assert(schemaChanges.removedFields().isEmpty)
     assert(schemaChanges.updatedFields().isEmpty)
     assert(schemaChanges.addedFields().size() == 1)
-    assert(schemaChanges.addedFields().get(0) == fieldMappingAfter(2))
+    assert(schemaChanges.addedFields().get(0) == fieldMappingAfter.get("data"))
+  }
+
+  private def newSchema(tuple: (Int, StructField)*): StructType = {
+    val fields = tuple.map { case (id, field) =>
+      addFieldId(id, field)
+    }
+    val schemaWithIds = new StructType(fields.asJava)
+    schemaWithIds
+  }
+
+  private def addFieldId(id: Int, field: StructField) = {
+    val metadataWithFieldIds = FieldMetadata.builder().fromMetadata(field.getMetadata)
+      .putLong("delta.columnMapping.id", id)
+      .putString("delta.columnMapping.physicalName", s"col-$id")
+      .build()
+    field.withNewMetadata(metadataWithFieldIds)
   }
```

Comment:
nit - can you move these to be next to other helpers in the test suite? just for better org, kind of weird to have them in the middle of the tests

Reply:
done.